src/Core/src/Platform/Android/PickerManager.cs

```diff
@@ -18,25 +18,14 @@ internal static class PickerManager
 		public static void Init(EditText editText)
 		{
 			editText.Focusable = true;
+			editText.FocusableInTouchMode= false;
```

Comment:
would it be possible to have the datepicker and time picker also call this code? 

someway that we can make sure all three are always working the same?